actors/builtin/power/power_state.go

```diff
@@ -277,6 +285,7 @@ func (st *State) deleteDetectedFault(s adt.Store, a addr.Address) error {
 	if nominalPower.GreaterThanEqual(ConsensusMinerMinPower) {
 		// just regained a miner > min size
 		st.NumMinersMeetingMinPower++
+		st.TotalNetworkPower = big.Add(st.TotalNetworkPower, claim.Power)
```

Comment:
If total network power is to be independent of the min miner sizes (i.e. option (b)) this should be outside the if on 285.